src/services/zip-tax.ts

```diff
@@ -73,7 +72,20 @@ export default class ZipTaxService extends AbstractTaxService {
   }
 
   private buildAddressString(address: Address) {
-    return `${address.address_1} ${address.address_2}, ${address.city} ${address.province}, ${address.postal_code}`;
+    return Object.entries(address).reduce(
```

Comment:
I guess Object.entries automatically arranges keys in alphabetical order?

Reply:
i don't think it does? i thought this would work because the data is shaped like an `Address`, which has them in the correct order